### PR TITLE
Reorder README & add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by heavyrain266**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+We currently expect most models **to not load properly**.
+We cannot test how `bevy_mod_fbx` handles the export formats of every software out there.
+If your model loads properly, thank your lucky start,
+if you encounter any issue please do the following:
+
+- Try opening the model using the `scene_viewer` (run `cargo run --example scene_viewer --release -- /path/to/file.fbx`)
+  (if your file has textures, make sure to enable the correct file formats using `--features bevy/png bevy/jpg bevy/tga` etc.)
+- If it fails, open an [issue] on our Github repo with a screenshot in the scene viewer
+  and a screenshot of how the model should look like
+- Ideally provide a download link to your FBX model
```
Comment (nicopap) [inline CONTRIBUTING.md:15]: Probably worth adding a issue template as well <https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository>

File: README.md [hunk 2]
```diff
@@ -2,12 +2,20 @@
 
 Autodesk Filmbox (*.fbx) loader for Bevy Engine.
 
+<!-- TODO: ### Features -->
```
Comment (nicopap) [inline README.md]:
```suggestion
### Features

Current features:

- Loads meshes, textures and some materials properties
  (including diffuse texture, normal maps and occlusion maps)
- Supports Maya PBR material
- Supports scene tree transform hierarchy

**Planned** features:

- skeleton rig imports
- `bevy_animation` support
- An optional Phong/Lambert shader implementation
  for the most common type of FBX materials
- More compatibility (`IndexToDirect`, handle file axis properties,
  cameras and lights, N-gon triangulation)
```

**Review by nicopap — CHANGES_REQUESTED** [review]
I'd just like to see a feature list in the main README.md

File: CONTRIBUTING.md (hunk 1 above)
Comment (heavyrain266) [reply CONTRIBUTING.md:15]: I think so

File: README.md (hunk 2 above)
Comment (heavyrain266) [reply README.md]: Phong is not a material, it's interpolation technique, shading model & reflection model used for PBR/BRDF.

File: README.md [hunk 3]
```diff
@@ -2,12 +2,40 @@
 
 Autodesk Filmbox (*.fbx) loader for Bevy Engine.
 
+### Features
+
+- Load meshes, textures & material properties
+- Supported material properties:
+  - normal maps
+  - occlusion maps
+  - texture diffusion
```
Comment (nicopap) [inline README.md]:
```suggestion
  - diffuse texture
```
diffuse texture is just the name for the base color texture.

**Review by nicopap — APPROVED** [review]
Just one little tinny change

**Pull request merged**